javaparser-core/src/main/java/com/github/javaparser/ast/Modifier.java

```diff
@@ -1,73 +1,217 @@
+        return new Modifier(Keyword.SYNCHRONIZED);
+    }
+
+    public static Modifier nativeModifier() {
+        return new Modifier(Keyword.NATIVE);
+    }
+
+    public static Modifier strictfpModifier() {
+        return new Modifier(Keyword.STRICTFP);
+    }
+
+    public static Modifier transitiveModifier() {
+        return new Modifier(Keyword.TRANSITIVE);
+    }
+
+    /**
+     * The Java modifier keywords.
+     */
+    public enum Keyword {
+
+        PUBLIC("public", false),
+        PROTECTED("protected", false),
+        PRIVATE("private", false),
+        ABSTRACT("abstract", false),
+        STATIC("static", false),
+        FINAL("final", false),
+        TRANSIENT("transient", false),
+        VOLATILE("volatile", false),
+        SYNCHRONIZED("synchronized", false),
+        NATIVE("native", false),
+        STRICTFP("strictfp", false),
+        TRANSITIVE("transitive", false),
+        DEFAULT("", true);
+
+        private final String codeRepresentation;
+
+        private final boolean pseudoKeyword;
+
+        Keyword(String codeRepresentation, boolean pseudoKeyword) {
+            this.codeRepresentation = codeRepresentation;
+            this.pseudoKeyword = pseudoKeyword;
+        }
+
+        /**
+         * @return the Java keyword represented by this enum constant.
+         */
+        public String asString() {
+            return codeRepresentation;
+        }
+
+        /**
+         * @return true when this keyword is not an actual keyword in source code.
+         * The only case is "DEFAULT."
+         */
+        public boolean isPseudoKeyword() {
+            return pseudoKeyword;
+        }
+    }
+
+    private Keyword keyword;
+
+    public Modifier() {
+        this(Keyword.PUBLIC);
+    }
+
+    @AllFieldsConstructor
+    public Modifier(Keyword keyword) {
+        this(null, keyword);
+    }
+
+    /**
+     * This constructor is used by the parser and is considered private.
+     */
+    @Generated("com.github.javaparser.generator.core.node.MainConstructorGenerator")
+    public Modifier(TokenRange tokenRange, Keyword keyword) {
+        super(tokenRange);
+        setKeyword(keyword);
+        customInitialization();
+    }
+
+    @Override
+    @Generated("com.github.javaparser.generator.core.node.AcceptGenerator")
+    public <R, A> R accept(final GenericVisitor<R, A> v, final A arg) {
+        return v.visit(this, arg);
+    }
+
+    @Override
+    @Generated("com.github.javaparser.generator.core.node.AcceptGenerator")
+    public <A> void accept(final VoidVisitor<A> v, final A arg) {
+        v.visit(this, arg);
+    }
+
+    @Generated("com.github.javaparser.generator.core.node.PropertyGenerator")
+    public Keyword getKeyword() {
+        return keyword;
+    }
+
+    @Generated("com.github.javaparser.generator.core.node.PropertyGenerator")
+    public Modifier setKeyword(final Keyword keyword) {
+        assertNotNull(keyword);
+        if (keyword == this.keyword) {
```

Comment:
sometimes we use the space before the parenthesis (which I prefer) sometimes we don't.

Reply:
This comes from the code generator which is always right, so the other thing is wrong :-)